Lib9c/Currencies.cs

```diff
@@ -0,0 +1,70 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Libplanet;
+using Libplanet.Assets;
+using Nekoyume.TableData;
+
+namespace Nekoyume
```

Comment:
In a new file, isn't it okay to use the `Lib9c` namespace? 🤔 (I don't have a strong opinion for this suggestion).

Reply:
Oh, thanks! I'll change.🙂